content/zh/docs/_index.md

```diff
@@ -0,0 +1,15 @@
+---
+title: 文档
+linkTitle: Docs
+menu: { main: { weight: 10 } }
+aliases: [/zh/docs/workshop/*]
```

Comment:
These aliases were for historical reasons, but those pages never existed in `zh` forms, so we don't need aliases here:

```suggestion
```

Reply:
FYI, I've gone ahead and committed this suggested change.